app/services/notification/AwsCredentials.scala

```diff
@@ -0,0 +1,14 @@
+package services.notification
+
+import javax.inject.Singleton
+
+import com.amazonaws.auth.profile.ProfileCredentialsProvider
+import com.amazonaws.auth.{AWSCredentialsProviderChain, DefaultAWSCredentialsProviderChain}
+
+@Singleton
+class AwsCredentials {
```

Comment:
A holder object for the Aws credential provider...it's being used in creating the `AmazonSimpleEmailServiceAsync`. Maybe this is not a very good way to do this?!
Thought to keep it in a separate place (rather that embedding it in the code creating the `AmazonSimpleEmailServiceAsync`)  so that it can be re-used...?